lib/addresses/dao.json

```diff
@@ -0,0 +1 @@
+{"4":"0x95a7ce185efc2d1f13efd2a00ee9247c51ea7009","14945560":"0x183af3950364390a266edff2a0e7c4c2f95c0691","23827572":"0xe4e0e7fe54d9189df29a80c07ab733fc9a212761","42097210":"0x183af3950364390a266edff2a0e7c4c2f95c0691","51657314":"0xad3a80686847d55d7b14b930bc6db53681ae1b1e","55632786":"0x4fde16c57ddf6d4870d5edd599074bb50dc96f88","65047207":"0xb67567175ac213f6f622b23d3d972d5b877c4813","74971415":"0x183af3950364390a266edff2a0e7c4c2f95c0691","85406806":"0x192db9a1f757c7c2afd03cb5afe883b27d0d3937"}
```

Comment:
I am wondering how I can prevent that those local address details get commited. 
Those "14945560" network IDs are devchain IDs. 
so far I don't think it is worth to write code for that and maybe simply try to ignore commiting those changes, but if somebody has an idea...?

Reply:
Maybe a git pre commit hook would work. I have never worked with git hooks...

Reply:
Yes, I think that could be a good solution. Git hooks are pretty easy to work with actually.

Reply:
hmm. the issue is locally you need those addresses in the file but in the repo we only need the commonly used ones. 
so what would the hook do? 

then it probably does not change that often...